src/ApiProblem.php

```diff
@@ -39,12 +39,20 @@ class ApiProblem implements \ArrayAccess
 {
 
     /**
-     * The content type for a HTTP response carrying
+     * The content type for a JSON based HTTP response carrying
      * problem details.
      *
      * @var string
      */
-    const CONTENT_TYPE = 'application/problem+json';
+    const CONTENT_TYPE_JSON = 'application/problem+json';
```

Comment:
Which might also be continue to be `CONTENT_TYPE` as JSON prolly is the default and more common choice.

Reply:
Likely, but I'd prefer to not privilege either format at present if it all possible.  I think just adding the two constants is enough, and let people do their own thing thereafter.